samples/apps/bookinfo/install_cluster_extension.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Script to configure an external machine for bookinfo demo.
+
+ISTIO_BASE=${ISTIO_BASE:-${GOPATH:-${HOME}/go}}
+ISTIO_IO=${ISTIO_IO:-${ISTIO_BASE}/src/istio.io}
+
+. $ISTIO_IO/istio/install/tools/istio_vm_common.sh
+
+VM=${VM:-vmextension}
+
+function installFiles() {
+    local VM=$1
+    # This only needs to be run once per cluster - but no harm in running it again.
+    istioInitILB
+    istioGenerateClusterConfigs
+
+    # TODO: download files from release build
+    # Copy from build until release files are out - will populate ISTIO_FILES with the .deb and helper files.
+    istioCopyBuildFiles
+
+    # Prepare the VM, run istio on the VM
+    istioBootstrapVM $VM
+
+    # Copy the details app to the VM
+    istioCopy $VM \
+      $ISTIO_IO/istio/samples/apps/bookinfo/src/details \
+      $ISTIO_IO/istio/samples/apps/bookinfo/cluster_extension.sh
+
+    istioRun $VM "./cluster_extension.sh install"
+
+
+}
+
+# Helper to create the raw VM. You can also use an existing VM, or create using the UI.
+# Currently uses gcloud - can be extended with other environments.
+function istioVMInit() {
+  # TODO: check if it exists and do "reset", to speed up the script.
```

Comment:
Rename to createGCloudVM

Reply:
How about remove this completely - and add a comment that user should create the VM using the UI or CLI ? It may be convenient as a function - but it's pretty slow.
The only reason I kept it is to make sure firewall on 22 is set, and a deb/ubuntu image is used.

Reply:
Yep. remove and we explain steps in doc.